test/factories/users.rb

```diff
@@ -1,6 +1,7 @@
 FactoryGirl.define do
   factory :user do
-    
+    provider "facebook"
```

Comment:
I think it might be better to have a user factory that doesn't call Facebook or at least call this one facebook_user or something like that and have a normal user that doesn't need Facebook